distance-calculator/charts/features/templates/broker_feature.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: nautilus.dellemc.com/v1alpha1
+kind: ProjectFeature
```

Comment:
@Bilna if this chart is capable of deploying project feature on its own , do we need to remove the step in documents of doing the same in UI ? 
Or do we say that whatever you are trying to do in UI, can be replaced with deployable charts and provide it as a separate document section either in repo README or in code-hub doc

Reply:
We can demo it in two ways. 
1. using the Helm 
3. directly using SDP UI

So, the documentation we can provide. Users can choose which way they want to install